configuration.properties

```diff
@@ -1,6 +1,7 @@
 browser=chrome
-url=https://qa.nextbasecrm.com/
-backup_url=https://qa.azulcrm.com/
```

Comment:
The conflict here is line 2 and 3. This is what is currently in the project.
Please change url to nextbase because it is working again, and if it goes down, then we can change the code to run the "backup_url"

2 url=https://qa.nextbasecrm.com/
3 backup_url=https://qa.azulcrm.com/